elixir/README.md

```diff
@@ -96,7 +96,8 @@ Now you can verify that it's working by connecting to a websocket:
 <details>
   <summary>Client</summary>
 
-```elixir
+```bash
+# Note: The token value below is an example.  The token value you will need is generated and printed out when seeding the database, as described earlier in the document.
```

Comment:
We need the same note for 2 other tokens in this doc section

Reply:
Good catch.  Updated those two sections with the same note.